### PR TITLE
[INLONG-4761][Manager] Optimize DataNode related business and write some unit tests

### LINKED ISSUE #4761: [Improve][Manager] Optimize DataNode related business and write some unit tests
### Description

Optimize DataNode related business and write ut

Including using validation, modifying annotations

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by leosanqing**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Fixes #4761 

### Motivation

- Missing unit tests
- The validation method is not used correctly
- Find some bugs

### Modifications

- Add addReq and UpdateReq interfaces for add and update validation
- Add unit test DataNodeControllerTest

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is already covered by existing tests, such as:
    - DataNodeControllerTest

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/common/AddReq.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+package org.apache.inlong.manager.common.pojo.common;
+
+import javax.validation.groups.Default;
+
+/**
+ * used for validate add request fields group
+ */
+public interface AddReq extends Default {
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/common/AddReq.java:25]: It is recommended to change the class name to `SaveRequest`.

File: inlong-manager/manager-test/src/main/resources/h2/data.sql [hunk 2]
```diff
@@ -17,4 +17,15 @@
...
-        '2022-06-05 08:58:06', 'inlong_init', 'inlong_init');
\ No newline at end of file
+        '2022-06-05 08:58:06', 'inlong_init', 'inlong_init');
+
+INSERT INTO `user` (`id`, `name`, `password`, `account_type`, `due_date`, `create_time`,
+                    `update_time`, `create_by`, `update_by`)
+VALUES (2, 'operator', '628ed559bff5ae36bd2184d4216973cf', 1, '2099-12-31 23:59:59', '2022-06-05 08:58:06',
+        '2022-06-05 08:58:06', 'inlong_init', 'inlong_init');
+
+INSERT INTO `data_node` (`id`, `name`, `type`, `url`, `username`, `token`, `ext_params`,
```
Comment (healchow) [inline inlong-manager/manager-test/src/main/resources/h2/data.sql:27]: Why insert this data rather than save it by code?

**Comment by dockerzhang** [comment]
@leosanqing please process the conflict firstly.

**Review by kipshi — APPROVED** [review]

**Review by ciscozhou — APPROVED** [review]

**Comment by leosanqing** [comment]
> @leosanqing please process the conflict firstly.

done

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**